.github/ISSUE_TEMPLATE/feature-request-fr.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Demande de fonctionnalité
+about: Suggérer une idée pour le projet
+title: "[FEATURE] Courte description de la fonctionnalité"
+labels: ''
+assignees: ''
+
+---
+
+## Information nécessaire
+
+**Est-ce que votre demande de fonctionnalité est reliée à un problème ? Veuillez élaborer.**
```

Comment:
```suggestion
**Est-ce que votre demande de fonctionnalité est reliée à un problème ? Veuillez élaborer.**
<!-- Mandatory -->
```

Reply:
Having it in the preceding section header is sufficient.